rs/replicated_state/src/canister_state/queues/input_schedule.rs

```diff
@@ -147,7 +147,7 @@ impl InputSchedule {
             InputQueueType::LocalSubnet => self.local_sender_schedule.pop_front(),
             InputQueueType::RemoteSubnet => self.remote_sender_schedule.pop_front(),
         }?;
-        debug_assert!(self.scheduled_senders.remove(&sender));
+        assert!(self.scheduled_senders.remove(&sender));
```

Comment:
Now we have a mix of debug asserts further up and asserts here. Is this what we want?

Reply:
Switched to a `remove()` call followed by a separate `debug_assert`. Now it's all `debug_asserts`.